addons/sale/data/mail_template_data.xml

```diff
@@ -96,7 +96,7 @@
         </table>
         <t t-foreach="object.order_line" t-as="line">
             <t t-if="(not hasattr(line, 'is_delivery') or not line.is_delivery) and line.display_type in ['line_section', 'line_note']">
-                <table width="100%" style="color: #454748; font-size: 12px; border-collapse: collapse;">
+                <table width="100%" style="color: #454748; font-size: 12px; border-collapse: collapse; white-space: nowrap;">
```

Comment:
so if I understand, if a product is too long, now the table will get out of the page 

> ![2024-01-30-114607_715x655_scrot](https://github.com/odoo/odoo/assets/9977887/f39c4cff-bc71-403c-89ec-4020c88e0ba9)
> 
![2024-01-30-114616_731x653_scrot](https://github.com/odoo/odoo/assets/9977887/a4e569ca-9af6-47f4-9250-cc0fcabfc8fc)

the ones on eg. "Total: 750$" seems ok to me, but for this part it seems worse to me

Reply:
![image](https://github.com/odoo/odoo/assets/55446255/5a3bbcd3-12eb-4e82-8169-bd2ca8812ee2)
The issue only occurs in chatter, not affecting the email itself. Here are before and after screenshots.

> if a product is too long, now the table will get out of the page

Yes, that's correct. However, I believe this ticket was created primarily due to the appearance of the Quantity, and only subsequently because of the items. 
Do you think this fix is unnecessary? 
Perhaps we should implement 'white-space: nowrap;' only for the title column and prices?

Reply:
what I meant by my comment was that I think the `white-space: nowrap;` could probably not be set on the part where there is the product name

and for "subtotal:" "taxes:" "total:" it would be ok to have the change

for the column header (Products/Quantity/Vat Excl.) this is ok too I think and can be kept too

if there was an issue for the quantity/vat excluded column cells (so [here](https://github.com/odoo/odoo/assets/9977887/0579b16e-ef61-4063-879d-cf8d4042534a)
) I guess it could be set on those `<td/>` if was also an issue.